src/main/java/com/gzs/main/RestMethods.java

```diff
@@ -17,13 +19,18 @@
     private ObjectMapper mapper;
     private TermDao termDao;
     private TranslationDao translationDao;
-    private Term term;
 
     public RestMethods() {
         mapper = new ObjectMapper();
-        termDao = new TermDaoImpl();
-        translationDao = new TranslationDaoImpl();
-        term = null;
+        Configuration config = new Configuration();
+        DaoFactory daoFactory = DaoFactory.getDAOFactory(config.getDataType());
+        termDao = daoFactory.getTermDao();
+        translationDao = daoFactory.getTranslationDao();
```

Comment:
@DespotOfMorea There is no reason to tie generation of test data to InMemoryDao. What if user wants to create test data for MySQLDao? Or what if user doesn't want to create test data for InMemoryDao? It should be both be possible.

I'd create another REST point which, when targeted will instantiate `GenerateTestData` and use the values to fill in the Dao (whichever it is, it doesn't care whether it is MySQL or InMemory).